net/tailscale/files/tailscale.init

```diff
@@ -23,6 +23,9 @@ start_service() {
   procd_open_instance
   procd_set_param command /usr/sbin/tailscaled
 
+  # starting with v1.48.1 ENV variable is required to enable autodetection of iptables / nftables
+  procd_set_param env TS_DEBUG_FIREWALL_MODE=auto
```

Comment:
It might be better to just set `nftables` here.

Detection is only supported for arm64 and amd64: https://github.com/tailscale/tailscale/blob/dc7aa98b768bf82017aa5cc82a62dd4d685f811d/util/linuxfw/linuxfw_unsupported.go#L4C58-L4C58

On MIPS this shows up:

```
router: detect iptables rule: linuxfw:unsupported
router: detect nftables rule: linuxfw:unsupported
router: nftables rule count: 0, iptables rule count: 0
router: using iptables
```

Reply:
> I wonder why it is not finding nftables?

The detection code only supports arm64 and amd64 architectures.

> Perhaps default should be nftables, and there can be a variable in [tailscale.conf](https://github.com/openwrt/packages/blob/master/net/tailscale/files/tailscale.conf) to go back to iptables.

I think this is the way to go.